app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="forms">
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
```

Comment:
- [ ] [OPTIONAL]
Generally, if a user not signed in, the user should be redirected to the login page, however, when enter the app it always assumes that the user is signed in. Let's endeavor to improve upon this aspect.

**For the first time app assumes the user is already signed in:**
![image](https://github.com/Atifzada/blog_app/assets/12745474/ac35775d-948a-481a-a8a9-9da0abf1cf3c)